go/vt/vtgate/vstream_manager_test.go

```diff
@@ -274,6 +274,74 @@ func TestVStreamChunks(t *testing.T) {
 	assert.Equal(t, int32(100), ddlCount.Get())
 }
 
+func TestVStreamManagerGetCells(t *testing.T) {
+	type testArgs struct {
+		name        string
+		optCells    string
+		cellAlias   bool
+		resultCells []string
+	}
+
+	tcases := []*testArgs{
+		{"default-local", "", false, []string{"aa"}},
+		{"default-local-cell-alias", "", true, []string{"local:aa", "region1"}},
+		{"with-opt-cells", "local:,bb,cc", true, []string{"local:aa", "bb", "cc"}},
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	cell := "aa"
+	ks := "TestVStream"
+	_ = createSandbox(ks)
+	hc := discovery.NewFakeHealthCheck(nil)
+	st := getSandboxTopo(ctx, cell, ks, []string{"-20", "20-40"})
+	vsm := newTestVStreamManager(hc, st, "aa")
+	ts, _ := st.GetTopoServer()
+
+	for _, tcase := range tcases {
+		t.Run(tcase.name, func(t *testing.T) {
+			vs := &vstream{
+				vgtid:              nil,
+				tabletType:         topodatapb.TabletType_PRIMARY,
+				optCells:           tcase.optCells,
+				filter:             nil,
+				send:               nil,
+				resolver:           nil,
+				journaler:          nil,
+				minimizeSkew:       false,
+				stopOnReshard:      true,
+				skewTimeoutSeconds: 0,
+				timestamps:         nil,
+				vsm:                vsm,
+				eventCh:            nil,
+				heartbeatInterval:  0,
+				ts:                 ts,
+			}
+
+			if tcase.cellAlias {
+				cellsAlias := &topodatapb.CellsAlias{
+					Cells: []string{"aa", "bb"},
+				}
```

Comment:
I think there's value in using different cell alias definitions too. i.e. making this a test case string slice variable rather than a bool.